### PR TITLE
move Search page to Dashboards category

### PR DESCRIPTION
**Pull request by irenerl24**
**What is this feature?**

This PR moves Search to the Dashboards category 

**Why do we need this feature?**

Currently that page is orphaned in our TOC

**Who is this feature for?**

All readers of documentation

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: docs/sources/dashboards/search/_index.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ labels:
 menutitle: Search
 title: Search
 weight: 80
+aliases:
+- /docs/grafana/latest/search/
```
[the remark this answers is not in the document]
Comment (irenerl24) [reply docs/sources/dashboards/search/_index.md]: Thanks so much for this!!

**Comment by imatwawana** [comment]
So maybe let's not merge this right away. The Search documentation only talks about dashboards, but I was looking at the actual search functionality and it applies to searching for other pages in the application and can drive actions like starting the configuration of an alert rule. I don't actually think moving this page makes sense. I think the content is just sorely lacking. It's a small feature, but I don't think the page is actually orphaned.

**Comment by irenerl24** [comment]
Thank you so much for your feedback, Isabel, much appreciated!

**Review by imatwawana — COMMENTED** [review]
I realize my confusion comes from the fact that the last section of this page talks about a global feature, the command palette, and that's what I've assumed this whole page was about all this time (clearly I never read it!). So I did some digging.

This page originally did live under **Dashboards** and didn't have any command palette content. It moved around a bit and eventually ended up outside of **Dashboards** even though all the content on the page was just about searching for dashboards. I think that may have been a mistake. What complicated matters was the addition of this global search content in v9.5. I think there may have been some confusion because the menuTitle of the page and H1 aren't the same. 

Anyhooo, I think what should probably happen here is this:

- Create a new page under **Dashboards** called **Search dashboards**
- Move all the content that's just about searching dashboards to that page
- Leave the command palette content on this existing standalone **Search** page
- This command palette content should eventually be made more robust when we have someone to cover that

**Comment by imatwawana** [comment]
> Totally agree with you here. This could be one of those nice first PRs for Jacob!

Excellent!

**Review by imatwawana — APPROVED** [review]

**Pull request merged**